lib/constants/common.ts

```diff
@@ -0,0 +1 @@
+export const MS_IN_S = 1_000;
```

Comment:
Should this not be `1000`?

When running this in node, i get `
SyntaxError: Invalid or unexpected token`

Reply:
i think you're right, that should be `1000`

Reply:
[typescript 2.7+](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-7.html#numeric-separators)  supports numeric separators